aiida_pseudo/groups/mixins/cutoffs.py

```diff
@@ -257,19 +250,20 @@ def get_cutoffs_unit(self, stringency: str = None) -> str:
         :raises ValueError: if no stringency is specified and no default stringency is defined for the family.
         :raises ValueError: if the requested stringency is not defined for this family.
         """
-        stringency = stringency or self.get_default_stringency()
+        if stringency is None:
+            stringency = self.get_default_stringency()
+        else:
+            self.validate_stringency(stringency)
 
         try:
             return self._get_cutoffs_unit_dict()[stringency]
-        except KeyError as exception:
+        except KeyError:
             # Workaround to deal with pseudo families installed in v0.5.0 - Set default unit in case it is not in extras
-            if stringency in self.get_cutoff_stringencies():
```

Comment:
do we no longer need this conditional?

Reply:
Isn't this already checked now by `validate_stringency`?

Reply:
Ah, I see now how this used to work. All good